packages/services-common/src/framework/lambda.ts

```diff
@@ -79,10 +79,14 @@ export const http = <T>(fn: (request: Request) => Promise<Response>) => async (
 
   const request = {
     method: event.httpMethod.toLocaleLowerCase(),
-    headers,
+    headers: { ...headers, ...event.multiValueHeaders },
     params: event.pathParameters,
     payload: body,
-    query: event.queryStringParameters || {},
+    query:
+      {
+        ...event.queryStringParameters,
+        ...event.multiValueQueryStringParameters,
+      } || {},
```

Comment:
```suggestion
      },
```

Since we are already creating an object no need to fallback.